core/src/main/java/hudson/cli/SetBuildResultCommand.java

```diff
@@ -33,7 +33,9 @@
  * Sets the result of the current build. Works only if invoked from within a build.
  * 
  * @author Kohsuke Kawaguchi
+ * @deprecated Limited to Remoting-based protocol.
  */
+@Deprecated
```

Comment:
See [**Risks and Assumptions**](https://gist.github.com/jglick/9721427da892a9b2f75dc5bc09f8e6b3#risks-and-assumptions) for impact on `distfork`.